src/main/java/com/philemonworks/selfdiagnose/SelfDiagnoseServlet.java

```diff
@@ -151,6 +174,8 @@ public DiagnoseRunReporter getReporter(String format) {
             return new XMLReporter();
         if ("json".equals(format))
             return new JSONReporter();
+        if ("http".equals(format))

```

Comment:
why did you choose to have 2 query parameters, status and http?

Reply:
removed.